app/services/search/suggestions_builder.rb

```diff
@@ -0,0 +1,28 @@
+class Search::SuggestionsBuilder
+  RADIUS_OPTIONS = [1, 5, 10, 15, 20, 25, 30, 35, 40, 45, 50, 55, 60, 70, 80, 90, 100, 200].freeze
+
+  attr_reader :radius, :radius_suggestions, :search_params
+
+  def initialize(search_params, radius)
+    @radius = radius.to_i
+    @search_params = search_params
+    get_radius_suggestions
+  end
+
+  def get_radius_suggestions
+    radius_idx = RADIUS_OPTIONS.find_index(radius)
+    wider_radii = (1..5).map { |idx| RADIUS_OPTIONS[radius_idx + idx] } unless radius_idx.nil?
```

Comment:
Maybe turn `unless radius_idx.nil?` into something more of a guard clause, because it is logic that wider_radii isn't really concerned with. I like the pattern where if a necessary variable is invalid, you stop running the method. Maybe it also avoids needing the `&.`s.